tests/unit/django_transform/test_fields.py

```diff
@@ -58,3 +62,44 @@ def filter_func(e):
     # Then
     object_to_serialize.my_attribute.all.assert_called_once()
     assert serialized_data == [3, 4]
+
+
+def test_django_feature_state_related_manager_field_serialize_discards_old_versions():
```

Comment:
should we add one more test where the live from is in future and that fs is no returned?

Reply:
Done 👍